### PR TITLE
Docs for IAMMETER devices

### PR DESCRIPTION
**Pull request by lewei50**
**Description:**
doc for IAMMETER devices

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#30465

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by frenck** [comment]
I'm not sure why you keep closing PR's and opening new ones.
There should be no reason for that.

~~Besides that, I've reviewed the previous one. So I'm going to close this PR and please continue on #11611~~ Nevermind, you've even deleted the upstream fork. In between.

This is now the 5 or 6th PR's for the same thing. I'm not reviewing this until I know you are not doing this again.

Thanks.

**Comment by lewei50** [comment]
@frenck sorry for open/close this PR,it's the first time I commit my code to github free project,I will not do open/close from now on.
I am very sorry for this.

**Comment by lewei50** [comment]
Hi , @frenck 
how are you.
In the past, due to my repeated close / open PR error operation, I have wasted a lot of time.
This is the first time I have submitted home assistant code. I want to make sure.
Do I just need to wait for the result of the review? Is there any step stuck because of my wrong operation now?
Thank you very much for your help.

**Comment by frenck** [comment]
I'm awaiting initial reviews on the codebase PR.

**Comment by lewei50** [comment]
> I'm awaiting initial reviews on the codebase PR.

should I just wait or need to do something ?

**Comment by frenck** [comment]
Just wait for a reviewer. We have a lot of things in the pipeline, it might take a bit.

File: source/_integrations/iammeter.markdown [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+description: "Instructions on how to integrate IAMMETER sensor within Home Assistant."
+logo: iammeter-logo.png
+ha_category:
+  - Energy
+  - Sensor
+ha_release: 0.104
+ha_iot_class: Local Polling
+---
+
+[IAMMETER](https://www.iammeter.com) is cloud energy management platform. Our Bi-Directional Wi-Fi Energy Meters can measure the real-time reading and report to IAMMETER server. These meters can also be easily integrated into home assistant system. We have both single phase (WEM3080, WEM3162) and three phase (WEM3080T) meters.
```
Comment (mrand) [inline source/_integrations/iammeter.markdown]:
```suggestion
`iammeter` provides real-time reading of single phase (WEM3080, WEM3162) and three phase (WEM3080T) meters from [IAMMETER](https://www.iammeter.com) over Wi-Fi.
```

File: source/_integrations/iammeter.markdown [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+## Configuration
+
+Use the meter's sensors in your installation, and add the following to your configuration.yaml file:
```
Comment (mrand) [inline source/_integrations/iammeter.markdown]:
```suggestion
To use this sensor in your installation, add the following to your `configuration.yaml` file:
```

**Review by frenck — APPROVED** [review]

**Pull request merged**